iot-cc-ranked/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -x
+
+docker build -t iot-cc-ranked:latest .
```

Comment:
How does `latest` here jive with the image versioning used in `helium-foundation-k8s`?

https://github.com/helium/helium-foundation-k8s/blob/55bf33c081ea73863a262b7a10f357822cb2c92f/manifests/poc-data-cluster/prod/helium/iot-cc-ranked.yaml#L25

Reply:
Fixed